packages/firestore/package.json

```diff
@@ -79,7 +78,8 @@
     "@firebase/app-types": "0.x"
   },
   "devDependencies": {
-    "@firebase/app": "0.6.14",
+    "@firebase/app": "0.6.13",
```

Comment:
bump back up to 14

Reply:
Yeah, I think this was causing all the build failures earlier.